edx2course_axis.py

```diff
@@ -51,113 +56,121 @@
 DATADIR = "DATA"
 
 VERBOSE_WARNINGS = True
-#FORCE_NO_HIDE = True
 FORCE_NO_HIDE = False
 
-# <codecell>
-
-#-----------------------------------------------------------------------------
+LOGGER = logging.getLogger()
```

Comment:
As discussed, you can do a per line disable with `# pylint: disable=invalid-name` or to keep the code cleaner you can add a `pylintrc` file with the following so that it applies to all uses:
```ini
[BASIC]
const-rgx = (([A-Z_][A-Z0-9_]*)|(__.*__)|log)$
```